package.json

```diff
@@ -55,6 +56,7 @@
     "migrate:create": "packages/backend-modules/migrations/script/create.js",
     "migrate:up": "DOTENV_CONFIG_PATH=apps/api/.env NODE_OPTIONS='-r dotenv/config' db-migrate up --migrations-dir packages/backend-modules/migrations/migrations",
     "migrate:down": "DOTENV_CONFIG_PATH=apps/api/.env NODE_OPTIONS='-r dotenv/config' db-migrate down --migrations-dir packages/backend-modules/migrations/migrations --verbose",
+    "heroku:migrate:up": "db-migrate up --migrations-dir packages/backend-modules/migrations/migrations",
```

Comment:
True, but it has been like this in the other migration commands. This might become a problem when we switch to pnpm eventually.

We could wrap db-migrate in a script inside the migrations package and only reference this script, but then again, the migrations package is also not explicitly listed as a workspace dependency, so we cannot be sure that the migrations package is actually there.